### DISCUSSION TITLE
Problem Installation on Linux Mint

### DISCUSSION THREAD

**Discussion by ramail0** [mytourbook/mytourbook#1274] [Q&A] [answered]
Hello, I struggle to get MT running on Linux Mint 21.2
Java is installed and the program is set to executable:
:~$ java --version
openjdk 11.0.21 2023-10-17
OpenJDK Runtime Environment (build 11.0.21+9-post-Ubuntu-0ubuntu122.04)
OpenJDK 64-Bit Server VM (build 11.0.21+9-post-Ubuntu-0ubuntu122.04, mixed mode, sharing)

[1706659689925.log](https://github.com/mytourbook/mytourbook/files/14105749/1706659689925.log)
The logfile says:
!SESSION 2024-01-31 01:08:09.730 -----------------------------------------------
eclipse.buildId=unknown
java.version=11.0.21
java.vendor=Ubuntu
BootLoader constants: OS=linux, ARCH=x86_64, WS=gtk, NL=de_DE
Command-line arguments:  -os linux -ws gtk -arch x86_64

!ENTRY net.tourbook 4 0 2024-01-31 01:08:10.487
!MESSAGE FrameworkEvent ERROR
!STACK 0
org.osgi.framework.BundleException: Could not resolve module: net.tourbook [10]
  Unresolved requirement: Import-Package: com.fasterxml.jackson.core.type
    -> Export-Package: com.fasterxml.jackson.core.type; bundle-symbolic-name="net.tourbook.ext.jars"; bundle-version="24.1.0.20240127-0827"; version="0.0.0"
       net.tourbook.ext.jars [30]
         Unresolved requirement: Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=17))"
  Unresolved requirement: Import-Package: com.fasterxml.jackson.databind
    -> Export-Package: com.fasterxml.jackson.databind; bundle-symbolic-name="net.tourbook.ext.jars"; bundle-version="24.1.0.20240127-0827"; version="0.0.0"

Is that a known issue? Help much appreciated.

**Comment by FJBDev** [comment, marked as answer]
@ramail0 
You need to install Java 17 to run MT 24.10, see the error from your log below

`Unresolved requirement: Require-Capability: [osgi.ee](http://osgi.ee/); filter:="(&([osgi.ee](http://osgi.ee/)=JavaSE)(**version=17)**)"`

But your Linux is using Java 11 at the moment
:~$ java --version
**openjdk 11.0.21 2023-10-17**

**Comment by ramail0** [comment]
Installed Openjdk-17-jre via software manager and first impression is that MT works fine. Thank you!
